ui/oopsyraidsy/oopsy_live_list.ts

```diff
@@ -272,9 +272,11 @@ export class OopsyLiveList implements MistakeObserver {
 
     const iconClass = m.type;
     const blame = m.name ?? m.blame;
-    const blameText = blame ? ShortNamify(blame, this.options.PlayerNicks) + ': ' : '';
+    const blameText = (blame !== undefined && blame.length > 0)
+      ? ShortNamify(blame, this.options.PlayerNicks) + ': '
+      : '';
     const translatedText = Translate(this.options.DisplayLanguage, m.text);
-    if (!translatedText)
+    if (translatedText === undefined || translatedText.length === 0)
```

Comment:
```suggestion
    if (translatedText === undefined)
```

The text is coming from the the trigger itself, and I think if it's weirdly blank we should still show a mistake line rather than special casing.